### PR TITLE
#21100 missing the server id when sending a PubSub message

### PR DESCRIPTION
**Pull request by jdotcms**
When sending the event out the server id was missing

### CODE REVIEW

**Review by nollymar — APPROVED** [review]

**Review by freddyucv — CHANGES_REQUESTED** [review]
any test here?

File: dotCMS/src/main/java/com/dotcms/dotpubsub/RedisPubSubImpl.java [hunk 1]
```diff
@@ -55,16 +67,17 @@ public void stop() {
     }
 
     @Override
-    public boolean publish(final DotPubSubEvent event) {
+    public boolean publish(final DotPubSubEvent eventIn) {
```
Comment (freddyucv) [inline dotCMS/src/main/java/com/dotcms/dotpubsub/RedisPubSubImpl.java:70]: doc?

File: dotCMS/src/main/java/com/dotcms/dotpubsub/RedisPubSubImpl.java [hunk 2]
```diff
@@ -17,12 +19,22 @@
  */
 public class RedisPubSubImpl implements DotPubSubProvider {
 
+    public final String serverId;
```
Comment (freddyucv) [inline dotCMS/src/main/java/com/dotcms/dotpubsub/RedisPubSubImpl.java:22]: it is public, is it ok or it should be private?

**Review by freddyucv — CHANGES_REQUESTED** [review]

**Review by erickgonzalez — APPROVED** [review]

**Comment by erickgonzalez** [comment]
Closing in favor https://github.com/dotCMS/core/pull/21329

**Pull request closed without merging**